src/spikeinterface/core/tests/test_sortinganalyzer.py

```diff
@@ -142,6 +142,27 @@ def test_SortingAnalyzer_tmp_recording(dataset):
         sorting_analyzer.set_temporary_recording(recording_sliced)
 
 
+def test_SortingAnalyzer_merge_all_extensions(dataset):
```

Comment:
I added these tests in the common extension tests. Core will not have all dependencies required by all extensions.